qiskit/aqua/operators/gradients/__init__.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Gradients (:mod:`qiskit.aqua.operators.gradients`)
+==================================================
+
+.. currentmodule:: qiskit.aqua.operators.gradients
```

Comment:
Can we possibly get some additional descriptive text here - you can see the sort of overview text that is in other parts of opflow. It would be nice to have some text here even to include some of that from the release notes where there is some overview and examples.

Reply:
Would it be too much to include everything from the release note?

Reply:
No I think that's probably even nice, the releasenotes are harded stumbled upon to than the docs so its nice to have them in a place where they are easily found 😉

Reply:
Whatever works given the timeframe etc. if most/all of the content of the release notes seems valid to have here then by all means.